File: 151004/Ivanenko/DC_Lab1/DC_Lab1/DC_Lab1/DTO/PostRequestTo.cs
using DC_Lab1.DTO.Interface;

namespace DC_Lab1.DTO
{
    public class PostRequestTo(int? TweetId, string? Content) : IRequestTo
    {
        public int Id { get; set; } = 0;
        public int? TweetId { get; set; } = TweetId;

        public string? Content { get; set; } = Content;
    }
}

File: 151002/Protchenko/lab1/rest-api/src/main/java/by/bsuir/restapi/model/dto/request/PostRequestTo.java
package by.bsuir.restapi.model.dto.request;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;

public record PostRequestTo(
        Long id,

        @NotNull
        Long issueId,

        @NotBlank
        @Size(min = 2, max = 2048)
        String content
) {
}
